src/EnergyPlus/HeatPumpWaterToWaterSimple.hh

```diff
@@ -67,15 +67,15 @@ namespace HeatPumpWaterToWaterSimple {
     struct GshpSpecs : public PlantComponent
     {
         // Members
-        std::string Name;                // user identifier
-        bool checkEquipName;             // name check flag
-        std::string WatertoWaterHPType;  // Type of WatertoAirHP ie. Heating or Cooling
-        int WWHPPlantTypeOfNum;          // equipment type num
-        bool Available;                  // need an array of logicals--load identifiers of available equipment
-        bool ON;                         // simulate the machine at it's operating part load ratio
-        bool IsOn;                       // flag that the heat pump is ON during current time step
-        bool MustRun;                    // flag that the heat pump is MUST RUN during current time step
-        Real64 SourceSideDesignMassFlow; // Design flow rate (kg/s)
+        std::string Name;                                 // user identifier
+        bool checkEquipName;                              // name check flag
+        std::string WatertoWaterHPType;                   // Type of WatertoAirHP ie. Heating or Cooling
+        DataPlant::PlantEquipmentType WWHPPlantTypeOfNum; // equipment type num
```

Comment:
Should not need to store the HPType as a string. Convert to enum when reading from IDF and just store that.